src/FormParserFactory.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace React\Http;
+
+use React\Http\Parser;
+
+class FormParserFactory
```

Comment:
Should this class be moved to the `Parser` namespace?
Also, perhaps consider renaming the namespace to be more descriptive?